### DISCUSSION TITLE
demo支持引less并将样式源码也展示在demo中吗

### DISCUSSION THREAD

**Discussion by zhongweiming** [umijs/dumi#2038] [Q&A]
能否像antd一样，demo里面引了less，在展示源码的时候就能看到引的less代码
![image](https://github.com/umijs/dumi/assets/23733295/3f22fa2f-ffa6-413b-8669-560c29daecad)
@PeachScript 各位大佬们看看

**Comment by PeachScript** [comment]
没看懂问题，现在就是支持展示 demo 引入的 less 文件源码的

**Reply by zhongweiming** [reply]
现在引入less是支持生效的，但是demo页面展示源码的时候看不到less源码呢。
![image](https://github.com/umijs/dumi/assets/23733295/25f669a0-a217-46e6-9a0e-c3148a3fa978)

**Reply by PeachScript** [reply]
我这里用脚手架复现不了，提供最小可复现仓库吧

**Reply by PeachScript** [reply]
有可能是组件库的样式把 dumi 的源代码 tab 干掉了，可以先自行排查下
